examples/azure-service-mesh/security/cert-manager-letsencrypt/readme.md

```diff
@@ -0,0 +1,276 @@
+# Cert-Manager Let's Encrypt Integration with Istio-based service mesh add-on
+
+This document contains instructions on how to integrate Istio-based service mesh add-on for AKS with cert-manager and obtain let's encrypt certificates for setting up secure ingress gateways. 
```

Comment:
Should we call out support policy about third-party integrations with cert-manager and LetsEncrypt, and K8s ingress, being allowed but unsupported? @shashankbarsin @biefy

Reply:
Or is that already implied by having it in the examples dir?

Reply:
this has already been been addressed as part of a note under `Create the shared configmap` section.
@nshankar13

Reply:
I saw - left a comment there.